runtime/jcl/CMakeLists.txt

```diff
@@ -23,6 +23,14 @@
 ###
 ### Set up shared interface libraries
 ###
+set(JCLS
+	se7_basic
+	se9
+	se9_before_b165
+	se10
+	se11
```

Comment:
`se12` should be added to this list since https://github.com/eclipse/openj9/pull/3638.